index.html

```diff
@@ -0,0 +1,216 @@
+                src="./assets/images/project.png"
+                alt="project 2"
+              />
+            </li>
+            <li>
+              <img
+                class="completed-projects-section-content-img"
+                src="./assets/images/project.png"
+                alt="project 3"
+              />
+            </li>
+            <li>
+              <img
+                class="completed-projects-section-content-img"
+                src="./assets/images/project.png"
+                alt="project 4"
+              />
+            </li>
+            <li>
+              <img
+                class="completed-projects-section-content-img"
+                src="./assets/images/project.png"
+                alt="project 5"
+              />
+            </li>
+            <li>
+              <img
+                class="completed-projects-section-content-img"
+                src="./assets/images/project.png"
+                alt="project 6"
+              />
+            </li>
+            <li>
+              <img
+                class="completed-projects-section-content-img"
+                src="./assets/images/project.png"
+                alt="project 7"
+              />
+            </li>
+            <li>
+              <img
+                class="completed-projects-section-content-img"
+                src="./assets/images/project.png"
+                alt="project 8"
+              />
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="contact-section" id="contact-section">
+        <div class="contact-section-title">
+          <h4>Contact Me</h4>
+        </div>
+
+        <div class="contact-section-content">
+          <div class="contact-section-content-logo">
+            <div class="hoverwrap">
+              <img src="./assets/images/linkedin.png" alt="Linkedin profile" />
+              <div class="hovercap">linkedin.com/in/Alexandru--Tanase</div>
+            </div>
+          </div>
+
+          <div class="contact-section-content-logo">
+            <div class="hoverwrap">
+              <img src="./assets/images/ymail.png" alt="Yahoo Mail" />
+              <div class="hovercap">t_alexandru87@yahoo.com</div>
+            </div>
+          </div>
+          <div class="contact-section-content-logo">
+            <div class="hoverwrap">
+              <img src="./assets/images/gmail.png" alt="Gmail" />
+              <div class="hovercap">talex21101987@gmail.com</div>
+            </div>
+            
+          </div>
+          <div class="contact-section-content-logo">
+            <div class="hoverwrap">
+              <img src="./assets/images/github.png" alt="GitHub" />
+              <div class="hovercap">talexandru1987</div>
+            </div>
+          </div>
+        </div>
+      </section>
+      
+    </main>
+    <footer>
+      <p class="footer-title">Made by Alexandru</p>
+    </footer>
+  </body>
+</html>
```

Comment:
This is all really good. Good use of semantic elements and all really clean code. Well done.